js-sdkTests/signTxn.js

```diff
@@ -31,8 +21,19 @@ const go = async () => {
     litNetwork: "serrano",
   });
   await litNodeClient.connect();
+  
+  // you need an AuthSig to auth with the nodes
+  // normally you would obtain an AuthSig by calling LitJsSdk.checkAndSignAuthMessage({chain})
+  // NOTE: to replace with a new one that you get from oauth-pkp-signup-example
+  const authSig = {
```

Comment:
Why are we changing the AuthSig and taking out SIWE resources, etc?

Reply:
The original auth sig was expired, and throwing invalid signature recovery errors on the node, this was generated to pass the auth as that wasn't the objective of the tests. 

We can make a signature generator for the test register the address to the PKP.